xml/admin_monitoring_alerting.xml

```diff
@@ -71,8 +71,8 @@
  <para>
   By default, traffic to &grafana; is encrypted with TLS. You can either supply
   your own TLS certificate or use a self-signed one. A self-signed certificate
-  is automatically created and configured for Grafana if no custom certificate
-  has been configured before Grafana has been deployed.
+  is automatically created and configured for &grafana; if no custom certificate
+  has been configured before &grafana; has been deployed.
```

Comment:
As lines in between have been removed, I cannot make a `suggestion`. So:

This specifies  what happens before the conditions that might cause it to happen. That is not ideal.

I suggest:

«
If no custom certificate has been configured before &grafana; has been deployed,
then a self-signed certificate is automatically created and configured for &grafana;.
»